client.go

```diff
@@ -118,3 +118,73 @@ func newClientUsingAddresses(port uint16, remoteAddresses []string) (*Client, er
 		return newClientUsingCluster(cluster)
 	}
 }
+
+// Fetch is a simple invocation of the NewFetchValueCommand
+func (c *Client) Fetch(bucketType, bucket, key string) ([]*Object, error) {
+	cmd, err := NewFetchValueCommandBuilder().
+		WithBucketType(bucketType).
+		WithBucket(bucket).
+		WithKey(key).
+		Build()
+
+	if err != nil {
+		return nil, err
+	}
+	if err = c.cluster.Execute(cmd); err != nil {
+		return nil, err
+	}
+
+	res := cmd.(*FetchValueCommand)
+	if res.Error() != nil || res.Response.IsNotFound {
+		return nil, res.Error()
+	}
+
+	return res.Response.Values, nil
+}
+
+// Store is a simple invocation of the StorehValueCommand
+func (c *Client) Store(bucketType, bucket, key string, obj *Object) ([]*Object, error) {
+	cmd, err := NewStoreValueCommandBuilder().
+		WithBucketType(bucketType).
+		WithBucket(bucket).
+		WithKey(key).
+		WithContent(obj).
+		Build()
+
+	if err != nil {
+		return nil, err
+	}
+	if err = c.cluster.Execute(cmd); err != nil {
+		return nil, err
+	}
+
+	res := cmd.(*StoreValueCommand)
+	if res.Error() != nil {
+		return nil, res.Error()
+	}
+
+	return res.Response.Values, nil
+}
+
+// Delete is a simple invocation of the NewFetchValueCommand
+func (c *Client) Delete(bucketType, bucket, key string) error {
```

Comment:
Add a `vclock []byte` argument. It doesn't *have* to be set but if your bucket has `allow_mult` set to true (default for 2.0 buckets) you'll need a vclock to prevent siblings.